notebooks/embeddings/ACP_prose_embeddings.ipynb

```diff
@@ -210,17 +210,17 @@
      "output_type": "stream",
      "text": [
       "# loading the pre-trained ProSE DLM model\n",
-      "# writing: ../data/acp/prose/train/acp_train_dlm_avg.h5\n",
+      "# writing: ../../data/acp/prose/train/acp_train_dlm_avg.h5\n",
       "# embedding with pool=avg\n",
-      "# 1361 sequences processed...\r"
+      "# 1372 sequences processed...\r"
```

Comment:
Small question: do you know why 11 more sequences are processed than before this commit? And why it's not 1378, the total number of sequences in ACP train?

Reply:
Ah I see, makes sense, thanks!